src/app/Http/Controllers/API/AuthController.php

```diff
@@ -106,4 +106,57 @@ public function user(Request $request)
     {
         return new UserResource($request->user());
     }
+  
+  /**
+     * Send an email to the user with the reset password link
+     *
+     * @bodyParam email email required email of the user who makes the request
+     *
+     * @param Request $request
+     */
+    public function sendResetLinkEmail(Request $request)
+    {
+        $request->validate([
+            'email' => 'required|email|exists:users,email'
+        ]);
+
+        // generate and save the token
+        $token = Str::random(60);
```

Comment:
Only works after adding `use Illuminate\Support\Str;` as import, please verify whether it's needed or not.

Reply:
no need to import because of the namespace